app/pandas.ipynb

```diff
@@ -549,14 +1071,53 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# write your code here\n"
+    "# write your code here\n",
+    "def unknown_signature():\n",
+    "    s = pd.Series(np.random.randint(0, 100, size=1000000))\n",
+    "    s.rolling(window=5).mean().dropna()\n",
```

Comment:
The Pandas solution for the moving average should return the calculated moving averages, not the original series. Modify the return statement to return the result of 's.rolling(window=5).mean().dropna()'.